addons/binding/org.openhab.binding.synopanalyzer/src/main/java/org/openhab/binding/synopanalyzer/handler/SynopAnalyzerHandler.java

```diff
@@ -61,6 +62,7 @@
 
     public SynopAnalyzerHandler(Thing thing) {
         super(thing);
+        SYNOP_DATE_FORMAT.setTimeZone(TimeZone.getTimeZone("UTC"));
```

Comment:
I would really like to go for a more Java 8 solution, especially since we are in control of time field to be formatted, with this extra change it becomes a little messy imho.

Some hints https://stackoverflow.com/a/26142960

If you can't figure it out I might have some time in the beginning of this weekend.